src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -63,13 +41,15 @@ internal FunctionInvokerBase(ScriptHost host, FunctionMetadata functionMetadata)
 
         public ScriptHost Host { get; }
 
+        public FunctionLogInfo LogInfo { get; }
+
         public FunctionMetadata Metadata { get; }
 
-        internal TraceWriter FileTraceWriter { get; set; }
+        protected TraceWriter TraceWriter => LogInfo.TraceWriter;
```

Comment:
Why is this and the next property internal? I don't see a subclass in this PR requiring this (maybe I missed it)